components/BottomNavigation/tests/unit/BottomNavigationTests.m

```diff
@@ -144,6 +144,46 @@ - (void)testAccessibilityIdentifier {
   XCTAssert([bar.itemViews.firstObject.accessibilityIdentifier isEqualToString:newIdentifier]);
 }
 
+-(void)testAccessibilityLabel {
+  NSString *oldLabel = @"oldLabel";
```

Comment:
Please break these into tests for single behaviors and add  "Given", "When", and "Then" comments to the tests.  An example might be:

```objc
- (void)testAccessibilityHintInitialValue {
  // Given
  NSString *oldHint = @"oldHint";
  NSString *newHint = @"newHint";
  UITabBarItem *tabBarItem = [[UITabBarItem alloc] initWithTitle:@"Home"
                                                           image:nil
                                                             tag:0];
  tabBarItem.accessibilityHint = oldHint;
  MDCBottomNavigationBar *bar = [[MDCBottomNavigationBar alloc] init];
  // When
  bar.items = @[ tabBarItem ];

  // Then
  XCTAssert([bar.itemViews.firstObject.accessibilityHint isEqualToString:oldHint]);
}

(void)testAccessibilityHintValueChanged {
  // Given
  NSString *oldHint = @"oldHint";
  NSString *newHint = @"newHint";
  UITabBarItem *tabBarItem = [[UITabBarItem alloc] initWithTitle:@"Home"
                                                           image:nil
                                                             tag:0];
  tabBarItem.accessibilityHint = oldHint;
  MDCBottomNavigationBar *bar = [[MDCBottomNavigationBar alloc] init];
  bar.items = @[ tabBarItem ];

  // When
  tabBarItem.accessibilityHint = newHint;

  // Then
  XCTAssert([bar.itemViews.firstObject.accessibilityHint isEqualToString:newHint]);
}

Reply:
Sounds good!  Went ahead and split the unit tests as suggested.